cds_books/config.py

```diff
@@ -135,16 +136,10 @@ def _parse_env_bool(var_name, default=None):
 CELERY_RESULT_BACKEND = 'redis://localhost:6379/2'
 #: Scheduled tasks configuration (aka cronjobs).
 CELERY_BEAT_SCHEDULE = {
-    'indexer': {
-        'task': 'invenio_indexer.tasks.process_bulk_queue',
-        'schedule': timedelta(minutes=5),
-    },
-    'accounts': {
-        'task': 'invenio_accounts.tasks.clean_session_table',
-        'schedule': timedelta(minutes=60),
-    },
+    **ILS_CELERY_BEAT_SCHEDULE,
```

Comment:
You can remove this if there is no change: `CELERY_BEAT_SCHEDULE` will be taken from ILS.